pytest-{{cookiecutter.plugin_name}}/pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = [
+  "setuptools>=61.0.0",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pytest-{{cookiecutter.plugin_name}}"
```

Comment:
We do not need any directive for setuptools to work with the `src` layout? Neat.

Reply:
Yes it has "automatic discovery" https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#automatic-discovery

I verified that the sdist and wheel look good (possible I missed something but I don't think so)